packages/modules/desktop-viewer-react/package.json

```diff
@@ -60,7 +60,7 @@
     "@types/react-redux": "^7.1.9",
     "concurrently": "^5.2.0",
     "copyfiles": "^2.1.0",
-    "electron": "^24.8.3",
+    "electron": "^24.8.5",
```

Comment:
coudnt you have just ran `rush update --full` to pick up new electron update instead of needing to cut a new patch ver of this pkg?

Reply:
Didn't know update --full can also do that. Reverted the changes in package.json for that package.